<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidgetTabela">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>QUADRADO PERFEITO</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>FATORIAL</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>NÚMERO PERFEITO</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>CAPICUA</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>NÚMERO PRIMO</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>BASE 10 PARA BASE 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>BASE 10 PARA BASE 8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>BASE 10 PARA BASE 16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>MÁXIMO DIVISOR COMUM</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>MÍNIMO MÚLTIPLO COMUM</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>PRIMOS ENTRE SI</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>NÚMERO 1    </string>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NÚMERO 2</string>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButtonExecutar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSegundoNumero">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPrimeiroNumero">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Primeiro Número</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Segundo Número</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
